labgrid/driver/networkusbstoragedriver.py

```diff
@@ -49,6 +51,52 @@ def on_activate(self):
     def on_deactivate(self):
         pass
 
+    @step(args=['filenames', 'target_dir', 'partition'])
+    def copy_files(self, filenames, target_dir, partition=1):
+        if not self.storage.path:
+            raise ExecutionError(
+                "{} is not available".format(self.storage_path)
+            )
+
+        dev_path = pathlib.PurePath(self.storage.path + str(partition))
+        mount_path = pathlib.PurePath('/media') / dev_path.name
+
+        mount_args = ["pmount", dev_path]
+        self.logger.debug('Mount {} to {}'.format(dev_path, mount_path))
+        try:
+            processwrapper.check_output(
+                self.storage.command_prefix + mount_args)
+        except subprocess.CalledProcessError as e:
+            # Proceed if device already has been mounted with pmount, otherwise
+            # give up
+            if not (e.returncode == 4 and str(mount_path) in str(e.output)):
```

Comment:
If the device is already mounted, we're in an unexpected case, and should probably abort?

Reply:
I agree, let us not make too many assumptions here. Removed.